src/login/styles/_Notifications.scss

```diff
@@ -43,6 +43,82 @@
   }
 }
 
+.notification-dot {
+  width: 12px;
+  height: 12px;
+  position: relative;
+  background-color: $orange-highlight;
+  border-radius: 50%;
+  margin-top: -1rem;
+}
+.notification-dot-inner{
+  background-color:$orange-highlight;
+  border: solid 1px $light-gray;
+  top: 10%; 
+  left: 10%;
+  width: 80%; /* of the notification-dot */
+  height: 80%; /* of the notification-dot */
+  position: absolute;
+  border-radius: 50%;
+}
+
+/* Wrapping */
+.speech-bubbletip-wrapper {
+  display: inline-block;
+  position: relative;
+}
+
+/* Absolute positioning */
+.speech-bubbletip {
+  display: inline-block;
+  position: absolute;
+  border-radius: 4px;
+  left: calc(50%);
+  -webkit-transform: translateX(-50%);
+  -ms-transform: translateX(-50%);
+      transform: translateX(-50%);
+  padding: 6px;
+  color: $white;
+  background: $black;
+  font-size: 0.875rem;
+  font-family: "ProximaNova-Regular", Arial, Helvetica, sans-serif;;
+  line-height: 1;
+  z-index: 100;
+  white-space: nowrap;
+}
+
+/* CSS border triangles */
+.speech-bubbletip::before {
+  content: " ";
+  border: solid transparent;
+  height: 0;
+  width: 0;
+  position: absolute;
+  pointer-events: none;
+  border-width: 6px;
+}
+
+/* Absolute positioning */
+.speech-bubbletip {
+  top: calc(50px * -1);
+  left: calc(50% * -1);
+}
+
+.speech-bubbletip.settings.en::before {
+  left: 52%;
+}
+
+/* CSS border triangles */
+.speech-bubbletip::before {
+  top: 100%;
+  border-top-color: $black;
+  left: 52%;
+}
+
+.speech-bubbletip.settings.sv::before {
+  left: 52%;
+}
+
```

Comment:
these lines seem to all have left 52%, so I would recommend combining them

Reply:
check, I will have a look